diff --git a/examples/offline_inference/vision_language_multi_image.py b/examples/offline_inference/vision_language_multi_image.py
index 43c44fa..d18c553 100644
--- a/examples/offline_inference/vision_language_multi_image.py
+++ b/examples/offline_inference/vision_language_multi_image.py
@@ -55,7 +55,7 @@ def load_aria(question, image_urls: List[str]) -> ModelRequestData:
 
 
 def load_deepseek_vl2(question: str, image_urls: List[str]):
-    model_name = "deepseek-ai/deepseek-vl2-tiny"
+    model_name = "deepseek-ai/deepseek-vl2-small"
 
     llm = LLM(model=model_name,
               max_model_len=4096,
@@ -479,7 +479,7 @@ if __name__ == "__main__":
     parser.add_argument('--model-type',
                         '-m',
                         type=str,
-                        default="phi3_v",
+                        default="deepseek_vl_v2",
                         choices=model_example_map.keys(),
                         help='Huggingface "model_type".')
     parser.add_argument("--method",
